cmd/kops-copy/main.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path"
+
+	"k8s.io/klog/v2"
+)
+
+func copyFile(source, target string) error {
+	klog.Infof("Copying source file %q to target directory %q", source, target)
+
+	sf, err := os.Open(source)
+	if err != nil {
+		return fmt.Errorf("unable to open source file %q: %w", source, err)
+	}
+	defer sf.Close()
+
+	fi, err := sf.Stat()
+	if err != nil {
+		return fmt.Errorf("unable to stat source file %q: %w", source, err)
+	}
+
+	fn := path.Join(target, path.Base(source))
```

Comment:
Nit: technically we should probably be using filepath when they are file paths, so it's one less thing to fix for windows...